ckanext/switzerland/i18n/fr/LC_MESSAGES/ckanext-switzerland.po

```diff
@@ -446,11 +446,21 @@ msgstr "Plus d'information"
 msgid "Private"
 msgstr "Privé"
 
+#: ckan/templates/package/snippets/package_basic_fields.html:98
+msgid "Public"
+msgstr "Publié"
+
 #: ckanext/switzerland/templates/package/read.html:14
 #: ckanext/switzerland/templates/snippets/package_item.html:40
 msgid "Draft"
 msgstr "Brouillon"
 
+msgid "Make draft"
+msgstr "Faire brouillon"
```

Comment:
hahaha, I read: `make bouillon` and now I want soup 🍵 😆

Reply:
:laughing: